README.md

```diff
@@ -158,8 +152,6 @@ ESLint works in a similar manner for all code linting.
 
 On a push to the repository or opening a pull request the [deploy.yml](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/.github/workflows/deploy.yml) file runs. This script sets up and does a number of things. For a simple push it's mostly checking code coverage.
 
```

Comment:
Nice to have would just be something that explains that playwright exists and is used in this repo. That said, it's beyond the scope of the ticket so take it or leave it.